seqr/utils/gene_utils.py

```diff
@@ -24,6 +24,10 @@ def get_genes_for_variant_display(gene_ids):
     return _get_genes(gene_ids, gene_fields=VARIANT_GENE_DISPLAY_FIELDS)
 
 
+def get_genes_for_prioritized_display(gene_ids):
```

Comment:
we want all the labels we show for SVS, I was just giving omim and constraint as examples. You should be loading the same data as for vartiants